lib/tasks/coveralls.rake

```diff
@@ -0,0 +1,2 @@
+require 'coveralls/rake/task'
+Coveralls::RakeTask.new
```

Comment:
Is there a specific reason you included this? From my reading of the docs it shouldn't be necessary.